tests/models/neural_networks/_build_static_data.py

```diff
@@ -0,0 +1,123 @@
+"""
+functions for calculating the static embedding
```

Comment:
This is only here for testing purposes while we decide where the best location is.

Do you think it should be a set of functions in ANALYSIS or as methods of the `ealstm` model? The static embedding is unique to the ealstm right so does it make sense there or in a new analysis class?